PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+Background
+-----
+
+Add background information that will be helpful to reviewers:
+  * Why are we making this change?
+  * How you arrived at this solution
+  * High level overview of when / how this code is executed
+
+Tasks
+-----
+* [ ] [Code of Conduct](https://github.com/Ibotta/atomic_cache/blob/master/CODE_OF_CONDUCT.md) reviewed
+* [ ] Specs written and passing
+* [ ] Backwards-incompatible changes called out in this PR
+* [ ] Increment the version file (`./lib/atomic_cache/version.rb`) when applicable
+    * See [semver.org](https://semver.org/) for what segment to increment
```

Comment:
Are lines 14 and 15 enough?  I'm a little concerned about the "when applicable" phrasing as it might not be obvious to everyone when/if this should happen.

Reply:
https://github.com/Ibotta/sopstool/blob/master/.github/PULL_REQUEST_TEMPLATE.md is slightly different - it has a separate versioning section